### ISSUE TITLE
Weird behaviour when PDB input files have no chainID

### ISSUE THREAD

**Issue by amjjbonvin** [haddocking/haddock3#138]
In the `topoaa` module, when an input PDB file has no chainID/segID defined the module puts `$PRO` as segid which causes problems downstream. No segIDs were defined in the config files since those are in principle defined in the `default.cfg` file of the module.

This does not happen when a PDB input has the chainID present. 

In `default.cfg` there are default segIDs defined for each molecule. Those should be used.

**Comment by joaomcteixeira** [comment]
Is it correct to say that each input PDB file must have a single chain?

**Comment by joaomcteixeira** [comment]
I am working on this right now.
If we have two input structures. One with chainID `A` and the other without chainID/segID (bugged). Which letter should we assign to the second structure? `A` or `B`? In other words, should the chainID/segID added to the structures with this information missing follow the IDs of the ones which have, or can they be independent?

**Comment by rvhonorato** [comment]
Since I was assigned first, I already fixed this :)

**Comment by rvhonorato** [comment]
> I am working on this right now. If we have two input structures. One with chainID `A` and the other without chainID/segID (bugged). Which letter should we assign to the second structure? `A` or `B`? In other words, should the chainID/segID added to the structures with this information missing follow the IDs of the ones which have, or can they be independent?

Basically needs to be the same as for 2.x

**Comment by rvhonorato** [comment]
I won't have any more time this week to clean and PR this one, so probably better if you go ahead @joaomcteixeira

**Comment by amjjbonvin** [comment]
In 2.4 the chains are assigned in the order of the input molecules (if not specified).

For 3.0 I would do the same, first molecule gets A, second B and so on, unless overwritten by parameters in the config file

**Comment by joaomcteixeira** [comment]
Will address this as part of a sanitization process for all PDBs entering the haddock3 pipeline.

**Comment by rvhonorato** [comment]
Would be nice to have this PDBHandler part as an independent library so we could also use it in the webserver;[ check what it does now]( https://github.com/haddocking/haddock-webserver-flask/blob/master/tools/PDBHandler.py).

**Comment by rvhonorato** [comment]
Is this one still relevant?

**Comment by joaomcteixeira** [comment]
Yes, it's relevant. It's solved in #144. But I need to finish that PR.

**Comment by rvhonorato** [comment]
Apparently it was fixed in #144
